src/www/ui/ui-export-list.php

```diff
@@ -678,6 +689,136 @@ private function printCSV($lines, $uploadtreeTablename, $copyright = false)
+        ],
+        'alignment' => [
+            'vertical' => \PhpOffice\PhpSpreadsheet\Style\Alignment::VERTICAL_CENTER,
+        ],
+        'borders' => [
+            'allBorders' => [
+                'borderStyle' => \PhpOffice\PhpSpreadsheet\Style\Border::BORDER_THIN,
+                'color' => ['argb' => \PhpOffice\PhpSpreadsheet\Style\Color::COLOR_BLACK],
+            ]
+        ],
+        'fill' => [
+            'fillType' => \PhpOffice\PhpSpreadsheet\Style\Fill::FILL_SOLID,
+            'startColor' => ['argb' => '002060'],
+        ],
+    ];
+    $sheet->getStyle('A1:J1')->applyFromArray($styleArray);
+    foreach ($headRow as $key => $val) {
+      $cellName = $key.'1';
+
+      $sheet->getColumnDimension($key)->setWidth($val[0]);
+
+      $sheet->setCellValue($cellName, $val[2]);
+    }
+
+    $styleArray = [
+        'font' => [
+            'color' => ['argb' => '808080'],
+        ],
+        'borders' => [
+            'vertical' => [
+                'borderStyle' => \PhpOffice\PhpSpreadsheet\Style\Border::BORDER_THIN,
+                'color' => ['argb' => 'B2B2B2'],
+            ]
+        ],
+        'fill' => [
+            'fillType' => \PhpOffice\PhpSpreadsheet\Style\Fill::FILL_SOLID,
+            'startColor' => ['argb' => 'FFFFCC'],
+        ],
+    ];
+    $sheet->getStyle('A2:J2')->applyFromArray($styleArray);
+    $annotationRow = array(
+      'A' => array('id', '-'),
+      'B' => array('path', '[Name of the Source File or Path]'),
+      'C' => array('name', '[Name of the OSS used]'),
+      'D' => array('version', '[Version Number of the OSS]'),
+      'E' => array('scanResults', '[Scan results. Use SPDX Identifier : https://spdx.org/licenses/]'),
+      'F' => array('concludedResults', '[Concluded results. Use SPDX Identifier : https://spdx.org/licenses/]'),
+      'G' => array('download', '[Download URL or a specific location within a VCS for the OSS]'),
+      'H' => array('homepage', '[Web site that serves as the OSSs home page]'),
+      'I' => array('copyright', '[The copyright holders of the OSS. E.g. Copyright (c) 201X Copyright Holder]'),
+      'J' => array('exclude', '[If this OSS is not included in the final version, Check exclude]'),
+      'K' => array('comment','')
+    );
+    foreach ($annotationRow as $key => $val) {
+      $cellName = $key.'2';
+
+      $sheet->setCellValue($cellName, $val[1]);
+    }
+
+    $styleArray = [
+        'borders' => [
+            'allBorders' => [
+                'borderStyle' => \PhpOffice\PhpSpreadsheet\Style\Border::BORDER_THIN,
+                'color' => ['argb' => \PhpOffice\PhpSpreadsheet\Style\Color::COLOR_BLACK],
+            ]
+        ],
+    ];
+
+    $id = 1;
+    foreach ($lines as $row) {
+      $rowNumber = $id + 2;
+      $range = 'A'.$rowNumber.':K'.$rowNumber;
+      $sheet->getStyle($range)->applyFromArray($styleArray);
+      $sheet->setCellValue('A'.$rowNumber, $id);
```

Comment:
Following change fixes the notice for me, however.
Can you please check @JustinWonjaePark ?
```suggestion
      $sheet->setCellValue('A'.$rowNumber, "$id");
```